README.md

```diff
@@ -113,6 +115,70 @@ const blogs = await mddb.getFiles({
 });
 ```
 
+# Configuring `mddb.config.js`
+
+## Overview
+
+`mddb.config.js` allows you to tailor the behavior of MarkdownDB by providing various configuration options. This section outlines the available options and how to use them.
+
+### 1. Integration of Configuration Options
+
+Integrate a configuration file (`mddb.config.js`) to enhance customization options for MarkdownDB.
+
+### 2. Computed Fields
+
+Implement computed fields to dynamically calculate values based on specified logic or dependencies.
+
+### 3. File Inclusion/Exclusion Patterns
+
+MarkdownDB supports configuring include and exclude file glob patterns, providing flexibility similar to the approach used in Next.js/TypeScript.
+
+### 4. Initializing Configuration Using CLI
+
+To initialize the MarkdownDB configuration file, use the following CLI command:
+
+```bash
```

Comment:
Generally i'd just copy and paste a file ...

Do we need a CLI to just generate a js file?